postgres/tests/test_statements.py

```diff
@@ -691,12 +709,16 @@ def test_activity_snapshot_collection(
         assert event['dbm_type'] == "activity"
         assert event['ddagentversion'] == datadog_agent.get_version()
         assert len(event['postgres_activity']) > 0
-        # find bob's query.
+        # find bob's query and blocking_bob's query
         bobs_query = None
+        blocking_bobs_query = None
```

Comment:
Oneliner suggestion for these searches: `next((q for q in event['postgres_activity'] if q.get('usename', None) == "bob"), None)`